single.php

```diff
@@ -13,9 +13,9 @@
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
-      $meta = get_post_meta($post->ID);
-      $photos = get_post_meta($post->ID, '_igv_photos');
-      $drawings = get_post_meta($post->ID, '_igv_drawings');
+    $meta = get_post_meta($post->ID);
+    $photos = get_post_meta($post->ID, '_igv_photos');
+    $drawings = get_post_meta($post->ID, '_igv_drawings');
```

Comment:
2 extra calls to get meta when meta is already in the `$meta` variable? I think this can all just go from that no?